packages/ddex/ingester/parser/ern38x.go

```diff
@@ -405,9 +424,8 @@ func buildSupportingTracks(release *common.Release) (tracks []common.TrackMetada
 		track.IsStreamFollowGated = parsedReleaseElem.IsStreamFollowGated
 		track.IsStreamTipGated = parsedReleaseElem.IsStreamTipGated
 		track.IsDownloadFollowGated = parsedReleaseElem.IsDownloadFollowGated
-		if !parsedReleaseElem.ReleaseDate.IsZero() {
-			track.ReleaseDate = parsedReleaseElem.ReleaseDate
-		}
+		track.ReleaseDate = parsedReleaseElem.ReleaseDate
```

Comment:
so there's no chance it would've been non-zero before and this sets it to the zero Time? i thought maybe this check was there because the ReleaseDate was already set

Reply:
track.ReleaseDate would be from the sound recording if I understand this correctly, and I don't see any release date logic when we [parse out the sound recording](https://github.com/AudiusProject/audius-protocol/blob/eca7b4e89d88320d7f7cef6d3638537b54906955/packages/ddex/ingester/parser/ern38x.go#L904) for a track, so it shouldn't be overriding anything, right?

Reply:
hm yeah looks like it would always be empty so i'm not sure why it was there. seems okay as is then